Packs/CarbonBlackEnterpriseEDR/Integrations/CarbonBlackEnterpriseEDR/CarbonBlackEnterpriseEDR.py

```diff
@@ -1291,6 +1336,85 @@ def process_search_get_command(client: Client, args: Dict) -> List[CommandResult
     return job_result_list
 
 
+def add_threat_tags_command(client: Client, args: Dict) -> CommandResults:
+    tags = args.get("tags")
+    threat_id = args.get("threat_id")
+    result = client.update_threat_tags(threat_id, tags)
+
+    readable_output = tableToMarkdown(f'Successfully updated threat: "{threat_id}" with tags "{tags}"', result, removeNull=True)
+    outputs = {
+        'ThreatID': threat_id,
+        'Tags': result.get('tags')
+    }
+
+    results = CommandResults(
+        outputs_prefix='CarbonBlackEEDR.Threat',
+        outputs_key_field='tags',
+        outputs=outputs,
+        readable_output=readable_output,
+        raw_response=result
+    )
+    return results
+
+
+def add_threat_notes_command(client: Client, args: Dict) -> CommandResults:
+    notes = args.get("notes")
+    threat_id = args.get("threat_id")
+    result = client.create_threat_notes(threat_id, notes)
+
+    readable_output = tableToMarkdown(f'Successfully added notes to threat: "{threat_id}"', result, removeNull=True)
+    outputs = {
+        'ThreatID': threat_id,
+        'Notes': notes
+    }
+
+    results = CommandResults(
+        outputs_prefix='CarbonBlackEEDR.Threat',
+        outputs_key_field='ThreatID',
+        outputs=outputs,
+        readable_output=readable_output,
+        raw_response=result
+    )
+    return results
+
+
+def add_alert_notes_command(client: Client, args: Dict) -> CommandResults:
+    notes = args.get("notes")
+    alert_id = args.get("alert_id")
+    result = client.update_alert_notes(alert_id, notes)
+
+    readable_output = tableToMarkdown(f'Successfully added notes to alert: "{alert_id}"', result, removeNull=True)
+    outputs = {
+        'AlertID': alert_id,
+        'Notes': notes
+    }
+
+    results = CommandResults(
+        outputs_prefix='CarbonBlackEEDR.Threat',
+        outputs_key_field='AlertID',
+        outputs=outputs,
+        readable_output=readable_output,
+        raw_response=result
+    )
+    return results
+
+
+def search_alerts_tags_command(client: Client, args: Dict) -> CommandResults:
+    tags = args.get("tags")
```

Comment:
```suggestion
    tags = argToList(args.get("tags"))
```
If possible give some tags